src/ng-select/ng-select.component.ts

```diff
@@ -113,6 +113,15 @@ export class NgSelectComponent implements OnInit, OnDestroy, OnChanges, AfterVie
     get single() {
         return !this.multiple;
     }
+    @Output('loadingChange') loadingChange: EventEmitter<Boolean> = new EventEmitter<Boolean>();
```

Comment:
Type `: EventEmitter<Boolean>` annotation can be removed as it is known from instance and will be automatically inferred by Typescript.